run_dir/design/clone_project.html

```diff
@@ -0,0 +1,83 @@
+{% extends "base.html" %}
+
+<!--
+Template file: clone_project.html
+URL: /clone_project
+Title: Clone an existing project in LIMS
+Description: Page to clone an existing project by pulling data from LIMS, modifying it and writing it back using LIMS API.
+-->
+
+{% block stuff %}
+<div class="container-xxl">
+  <div class="row">
+    <div class="col-10 offset-1">
+      <h1 class="page_title my-4">
+        <span id="page_title">Clone Project</span> <small class="text-muted">Clone an existing project in LIMS</small>
```

Comment:
Maybe add a bit more explanation on what it does and what the consequences are?

Reply:
Done?